src/project.js

```diff
@@ -0,0 +1,55 @@
+import ProjectStorage from './projectstorage';
+
+const Project = (name) => ({
+  name,
+});
+
+const localStorage = () => {
+  const projects = ProjectStorage().getProject();
+  if (projects !== null) {
+    return true;
+  }
+
+  return false;
+};
+
+const saveProject = (name) => {
+  const project = Project(name);
+  const ps = ProjectStorage();
+  if (localStorage()) {
+    const projects = ps.getProject();
+    projects.push(project);
+    ps.setProject(projects);
+  } else {
+    const tP = ProjectStorage().todoProjects;
+    tP.push(project);
+    ps.setProject(tP);
+  }
+};
+
+const validate = (name) => {
+  if (name === '') {
+    document.querySelector('.error-message').innerHTML = 'Name cannot be blank';
+  } else {
+    saveProject(name);
+  }
+};
```

Comment:
DOM-Manipulation and Logic are in separate modules

Reply:
@collinsugwu @abruzy  This was not attended to, how about creating a function that listens to the project title input field and passes the content to validate then displays that message only if it is blank